src/main/java/com/software/modsen/ratingservice/dto/request/DriverForRating.java

```diff
@@ -0,0 +1,3 @@
+package com.software.modsen.ratingservice.dto.request;
+
+public record DriverForRating(Long id) {}
```

Comment:
Я предпочитаю такую запись record-ов:
```suggestion
public record DriverForRating(
    Long id
) {
}
```